cpp/downgrades/25e365d1e8147df0f759b604f96eb4bffea48271/usings.ql

```diff
@@ -0,0 +1,17 @@
+class UsingEntry extends @using {
+  string toString() { none() }
+}
+
+class Element extends @element {
+  string toString() { none() }
+}
+
+class Location extends @location_default {
+  string toString() { none() }
+}
+
+from UsingEntry u, Element target, Location loc, int kind
+where
+  usings(u, target, loc, kind) and
+  kind != 3
+select u, target, loc
```

Comment:
Nothing in the database can refer to a `UsingEntry`, right?

Reply:
Fixed in https://github.com/github/codeql/pull/17006/commits/af562f15f3099a4291c281d85c1881fb401438d2. (Don't think we need to do anything on the upgrade side.)